common/src/main/java/com/scalar/dl/genericcontracts/table/v1_0_0/Constants.java

```diff
@@ -23,10 +43,22 @@ public class Constants {
       "The specified format of the indexes is invalid.";
   public static final String INVALID_RECORD_FORMAT =
       "The specified format of the record is invalid.";
+  public static final String INVALID_QUERY_FORMAT = "The specified format of the query is invalid.";
+  public static final String INVALID_CONDITION_FORMAT =
+      "The specified format of the condition is invalid. Condition: ";
+  public static final String INVALID_PROJECTIONS_FORMAT =
+      "The specified format of the projections.";
```

Comment:
I feel like this message is missing something. Should this be as follows?:

```suggestion
      "The specified format of the projections is invalid.";
```

Reply:
Good catch, thank you! Fixed in b8c8b9d.